Core/Core.cpp

```diff
@@ -188,6 +197,11 @@ void Core_RunLoop() {
 			Sleep(sleepTime);
 		if (!windowHidden) {
 			GPU_SwapBuffers();
+
+			// Keep the system awake for longer than normal for cutscenes and the like.
+			if (time_now_d() < lastActivity + ACTIVITY_IDLE_TIMEOUT) {
+				SetThreadExecutionState(ES_SYSTEM_REQUIRED | ES_DISPLAY_REQUIRED);
```

Comment:
Hmm, not sure.  It didn't seem to have any impact but I changed it to only happen every 89 seconds.

-[Unknown]